india_banking_connector/host_to_host/doctype/citi_h2h_connector/citi_sftp_client.py

```
@@ -0,0 +1,131 @@
+						"rsa-sha2-512",
+						"rsa-sha2-256",
+					],
+					"keys": [
+						"rsa-sha2-512",
+						"rsa-sha2-256",
+					],
+				},
+			)
+
+			self.sftp = self.client.open_sftp()
+			return True
```

Ruff (0.14.14)

[warning] Consider moving this statement to an `else` block

(TRY300)